tempest/api/workloadmgr/sanity/test_sanity_image_booted.py

```diff
@@ -35,45 +36,50 @@ def setup_clients(cls):
     def test_1_create_workload(self):
         try:
             #Prerequisites
+            global counter
             global vm_id
             global workload_id
             global snapshot_id
             global incr_snapshot_id
             self.created = False
-            
+            counter = 1

```

Comment:
Please make these changes as part of framework, instead of including in test script.

Reply:
you mean the counter variable? where should i add it? i mean which file in framework